test/simulator/timestamp/src/session_simulator.cpp

```diff
@@ -129,8 +134,21 @@ session_simulator::prepare_transaction(const std::string &config)
 
     auto pos = config_map.find("prepare_timestamp");
     if (pos != config_map.end()) {
+        WT_SIM_RET(ts_manager->validate_hex_value(pos->second, "prepare timestamp"));
+        uint64_t prepare_ts = ts_manager->hex_to_decimal(pos->second);
+        set_prepare_timestamp(prepare_ts);
     }
 
+    /* A prepared timestamp should have been set at this point. */
+    if (!has_prepare_timestamp())
+        WT_SIM_RET_MSG(EINVAL, "'prepare_transaction' - prepare timestamp is not set");
+
+    /* commit timestamp must not be set before transaction is prepared. */
```

Comment:
`commit` -> `Commit`

Reply:
Updated in [e3e29bb](https://github.com/wiredtiger/wiredtiger/pull/8386/commits/e3e29bba3019a6429b3fbb5835e83da23c9cc41b)